### DISCUSSION TITLE
Energy of PowerCell holded by grabber

### DISCUSSION THREAD

**Discussion by Bielecki** [colobot/colobot#1652] [Q&A] [answered]
Hello,
I'd like to write a program responsible for recharging both of Grabber `energyCell` and grabbed `PowerCell`, but I can't find anywhere documentation about PowerCell energy level (equivalent of `energyCell.energyLevel` from Houston Program in Tropika 3 mission). Am I missing something, or there is no possibility of reading that value?

**Comment by tomaszkax86** [comment, marked as answer]
You mean you want to check energy level of a battery your robot is holding? The held object can be referenced with `load` and its energy level can be referenced with `load.energyLevel`. You can use `message(load.energyLevel);` to test this.

Just make sure your robot is actually holding something, if the robot is not holding anything `load` will be `null` and trying to access `energyLevel` of `null` will cause an error.

**Reply by Bielecki** [reply]
Thank you for answering me. That's right, that's what I needed to know. Is `load` in the docs? I couldn't find it. I've seen it only in variables of running program (its super useful btw).

**Reply by ghost** [reply]
Yes, it's in the docs. It's in types -> object.
